src/auth/computeclient.ts

```diff
@@ -43,6 +53,7 @@ export class Compute extends OAuth2Client {
     // refreshed before the first API call is made.
     this.credentials = {expiry_date: 1, refresh_token: 'compute-placeholder'};
     this.serviceAccountEmail = options.serviceAccountEmail || 'default';
+    this.scopes = arrify(options.scopes);
```

Comment:
I admit I don't know how gaxios works handles params, so this may be correct. My thought was we needed:

```suggestion
    this.scopes = scopes.join(',');
```

As the metadata server takes `.../token?scopes=https://www.googleapis.com/auth/drive,...`

Reply:
Yup!  `gaxios` will automatically take collections there and join them in the way you laid out :)

Reply:
Amazing 💫